app/main.py

```diff
@@ -1,8 +1,38 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```

Comment:
Separate parameters for better readability:

```suggestion
    def __init__(self,
                 comfort_class: int,
                 clean_mark: int, 
                 brand: str) -> None:
```

Also in all other similar cases